api/src/certification/complementary-certification/infrastructure/repositories/complementary-certification-badge-repository.js

```diff
@@ -41,6 +41,7 @@ const findAttachableBadgesByIds = async function ({ ids }) {
   const badges = await knex
     .from('badges')
     .whereIn('badges.id', ids)
+    .and.where('badges.isCertifiable', true)
```

Comment:
Neatpick : dans [la documentation knex actuelle](https://knexjs.org/guide/query-builder.html#whereraw), ils utilisent toujours le format camelCase plutôt que la notation pointée qui semble "plus ancienne". J'aurais du coup tendance à suivre le mouvement de la lib.

```suggestion
    .andWhere('badges.isCertifiable', true)
```

Reply:
GG pour l'endroit trouvé pour l'implem' 💪🏻  !